File: ABC/python/062/B.py
import sys
H,W = map(int,input().split())
if H < 0 or H > 100 or W < 0 or H > 100:
    sys.exit()
pixeles = []
for I in range(H):
    pixeles.append(str(input()))

print("#"*(W+2))
for L in pixeles:
    print("#",end='')
    print(L,end='')
    print("#")
print("#"*(W+2))

File: ABC/ruby/062/B.rb
H,W = gets().chomp.split(' ').map { |n| n.to_i }
exit if !( 1 <= H && H <= 100 && 1 <= W && W <= 100 )
B = Array.new(H) { gets().chomp.split(' ') }
header_footer = Array.new(W+2) { '#' }

puts header_footer.join()
H.times { |n|
    puts ("##{B[n].join()}#")
}
puts header_footer.join()